packages/enterprise-alert/index.test.js

```diff
@@ -2,10 +2,16 @@ import { render } from '@testing-library/react'
 import EnterpriseAlert from './'
 
 describe('<EnterpriseAlert />', () => {
-  test('renders correctly', () => {
+  test('passes a provided className to the root element', () => {
+    const className = 'my-enterprise-alert'
+    const { container } = render(<EnterpriseAlert className={className} />)
+    const rootElem = container.firstChild
+    expect(rootElem).toHaveClass(className)
+  })
+
+  test('renders product theming and messaging as expected ', () => {
     const { container } = render(<EnterpriseAlert product="consul" />)
     const rootElem = container.firstChild
-    expect(rootElem).toHaveClass('g-enterprise-alert')
```

Comment:
Removed this duplicate assertion from tests, covered by an additional `className` test above.